src/components/Footer.tsx

```diff
@@ -0,0 +1,208 @@
+                style={{ paddingRight: "1.5em", height: "36px" }}
+              />
+            </a>
+            <Text style={{ color: "rgba(255, 255, 255, 0.65)" }}>
+              The open source smart legal contract stack
+            </Text>
+            <Link href="mailto:admin@accordproject.org">
+              <Text strong style={{ color: "rgba(255, 255, 255, 0.65)" }}>
+                admin@accordproject.org
+              </Text>
+            </Link>
+            <Link href="https://discord.com/invite/Zm99SKhhtA" target="_blank">
+              <Button
+                type="primary"
+                shape="round"
+                size="large"
+                style={{ padding: "2px 24px" }}
+              >
+                Join
+              </Button>
+            </Link>
+          </Space>
+        </Col>
+        {footerSections.map((section) => (
+          <Col span={3} key={section.title}>
+            <Space direction="vertical" size="middle">
+              <Text
+                strong
+                style={{
+                  color: "rgba(255, 255, 255, 0.65)",
+                  fontSize: "11px",
+                  letterSpacing: "0.1em",
+                }}
+              >
+                {section.title}
+              </Text>
+              {section.links.map((link) => (
+                <Link
+                  href={link.href}
+                  key={link.title}
+                  style={{ color: "white", fontSize: "15px" }}
+                >
+                  {link.title}
+                </Link>
+              ))}
+            </Space>
+          </Col>
+        ))}
+      </Row>
+      <Row justify="space-between" align="middle" style={{ marginTop: "80px" }}>
+        <Col>
+          <Text style={{ color: "rgba(255, 255, 255, 0.65)" }}>
+            copyright © {year} accord project &bull;{" "}
+            <Link
+              strong
+              href="https://accordproject.org/privacy"
+              target="_blank"
+              style={{ color: "rgba(255, 255, 255, 0.65)" }}
+            >
+              trademark policy
+            </Link>{" "}
+            &bull;{" "}
+            <Link
+              strong
+              href="https://accordproject.org/brand-assets"
+              target="_blank"
+              style={{ color: "rgba(255, 255, 255, 0.65)" }}
+            >
+              brand assets
+            </Link>
+          </Text>
+        </Col>
+
+        <Col>
+          <Space>
+            <Link
+              href="https://github.com/accordproject"
+              target="_blank"
+              style={{ color: "rgba(255, 255, 255, 0.65)" }}
+            >
+              <GithubOutlined style={{ fontSize: "17px" }} />
+            </Link>
+            <Link
+              href="https://twitter.com/AccordHQ"
+              target="_blank"
+              style={{ color: "rgba(255, 255, 255, 0.65)" }}
+            >
+              <TwitterOutlined style={{ fontSize: "17px" }} />
```

Comment:
X it is now :))

Reply:
yeah I know :) funny enough antd doesn’t have an updated icon for X yet. i guess.

didn't want to bring in a third party library for an icon too.

would definitely figure a way around.